docroot/modules/custom/cs_core/config/install/views.view.civictheme_test_table_mini_pager.yml

```diff
@@ -0,0 +1,202 @@
+langcode: en
```

Comment:
Duplicated table view to demo and test the mini pager - only 1 pager type per view!!